Cluster and README updates, add tutorial slides

milroy opened the pull request:
This PR adds support for changing the AWS EBS storage class, fixes a problem with spawning EC2 instances in AZs that differ from the EKS cluster, updates the README with details for changing the class, and adds the tutorial slides as a PDF.

vsoch commented on 2023-RADIUSS-AWS/JupyterNotebook/aws/eksctl-autoscale-config.yaml:
```diff
@@ -0,0 +1,76 @@
+# https://www.arhea.net/posts/2020-06-18-jupyterhub-amazon-eks
```
Oh wow we needed an entire new config? Sheesh. Just so I understand:

1. This config is for when I want to customize the storage class
2. The eksctl-config.yaml is for when I don't

But both are for autoscaling right?

vsoch replied:
Could you add a note in the README where we create the cluster with eksctl examples for creating both, and then the context under which you'd want to do that?

vsoch commented on 2023-RADIUSS-AWS/JupyterNotebook/aws/eksctl-config.yaml:
```diff
@@ -3,33 +3,11 @@ apiVersion: eksctl.io/v1alpha5
 metadata:
   name: jupyterhub
-  region: us-east-2
+  region: us-east-1
```
Do we want these changes to remain (or was it testing/should be reverted?) I notice that this is in us-east-1, but the first is us-east-2.

Ideally, we could just use one config, or have identical configs that just vary based on adding the storage class. I think if we come back to this next year (or later) it might be confusing distinguishing the two.

milroy replied:
After more consideration I don't think it's necessary to have the exact config we use to create the tutorial session in the repo. Plus it will create confusion for the attendees. I'll clean this up and let you know when you can look again.

milroy commented:
After discussion and additional analysis I realized I had borked the previous working configuration. I've fixed the error and cleaned up the commits and messages as well as added instructions for creating the tutorial configuration. @vsoch you can take another look.

vsoch reviewed (approved):
It looks great - thank you for your hard work on this, and I hope it wasn’t too much. I’m really impressed we accomplished so much in just a few days! 

I’m turning into a potato, but let’s touch bases tomorrow about a final test/tests before Thursday - I’d like to give the notebook one last look (probably easiest through local deploy) and also possibly give the flux maintainers a chance for a final A-OK. We can also do a final sanity check if the min/max values make sense for the instance type, concurrency limit, and faux user pods, etc.

LGTM!
